src/Components/MainTableView/DataStorageClasses/TableAttribute.tsx

```diff
@@ -55,6 +55,24 @@ class TableAttribute {
       return date.toDateString();
     }
 
+    /**
+     * Helper function for converting view purpose dateTime string back to datajoint dateTime format
+     * @param UTCdateTimeString 
+     */
+    static parseDateTimeToDJ(UTCdateTimeString: string) {
```

Comment:
Perhaps change name to parseDateTimeToDJFormat

Reply:
addressed